audb/core/load.py

```diff
@@ -1722,14 +1755,23 @@ def load_table(
             version,
         )
 
+        # Find misc tables used in schemes of the requested table
```

Comment:
This ensures to only load required misc tables I take it. 
At first it was a bit awkward that you below then iterate over the tables, 
and then do list wrapping `table` as `[_table]`again. 
Reading it a second time it makes sense. 

Would it make sense clarify the docstring and say 
`Find (only those) misc tables ... `

Reply:
I updated the comment using your suggestion.

It is indeed a little bit unfortunate that we need to use `_table` in the loop. `table` is basically the "requested table", but `requested_table` is not a good variable name, and `_table` is just the variable name for the iteration over all tables we need to download.

Reply:
Yes. This was only an observation as when reading the code you have to think twice.  Prefixing it with `_` is fair I think as it flags that this is only an iteration placeholder. I will resolve this conversation.